cloudferrylib/os/network/neutron.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, config, cloud):
         self.cloud = cloud
         self.identity_client = cloud.resources['identity']
         self.neutron_client = self.proxy(self.get_client(), config)
+        self.ext_net_map = utl.read_yaml_file(self.config.migrate.ext_net_map) \
+            if utl.read_yaml_file(self.config.migrate.ext_net_map) else {}
```

Comment:
`self.ext_net_map = utl.read_yaml_file(self.config.migrate.ext_net_map) or {}` would be much cleaner.